test/e2e/persistent_volumes.go

```diff
@@ -326,6 +351,37 @@ func createWaitAndDeletePod(f *framework.Framework, c *client.Client, ns string,
 	return nil
 }
 
+// validate PV/PVC, create and verify writer pod, delete PVC and PV. Ensure
+// all of these steps were successful.
+func completeTest(f *framework.Framework, c *client.Client, ns string, pv *api.PersistentVolume, pvc *api.PersistentVolumeClaim) (*api.PersistentVolume, *api.PersistentVolumeClaim, error) {
```

Comment:
I totally missed this one, could you please add some comment to make it more visible?